### ISSUE TITLE
ssd7 evaluation problem

### ISSUE THREAD

**Issue by JMinlj** [pierluigiferrari/ssd_keras#251]
Thank you for the work,I use ssd7 to train Caltech pedestrian dataset with parse_xml( )  ,  after 3 epochs  the loss is 4.6166 and val_loss is 3.7410.I want to know the  training effect，evaluating the model with ssd300_evaluation.py and modifying the model=ssd_300( ) to model=build_model of ssd7.The AP is 0.01,I don't know how to solve the problem, thank you if any help.
